src/main/java/servlets/admin/userManagement/DeletePlayers.java

```diff
@@ -0,0 +1,155 @@
+package servlets.admin.userManagement;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import org.apache.log4j.Logger;
+import org.owasp.encoder.Encode;
+
+import dbProcs.Getter;
+import dbProcs.Setter;
+import utils.ShepherdLogManager;
+import utils.Validate;
+import utils.UserKicker;
+
+/**
+ * The delete players server side functionality is hanled by this control class
+ * <br/><br/>
+ * This file is part of the Security Shepherd Project.
+ * 
+ * The Security Shepherd project is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.<br/>
+ * 
+ * The Security Shepherd project is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.<br/>
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with the Security Shepherd project.  If not, see <http://www.gnu.org/licenses/>. 
+ * @author Cosmin Craciun
+ *
+ */
+public class DeletePlayers extends HttpServlet{
```

Comment:
This method needs an associated JUnit Test for the Definition of Done

Reply:
Added associated JUnit test class src/test/java/servlets/admin/userManagement/DeletePlayersTest.java

Reply:
Looks Great!